src/pages/characters/[[...params]].js

```diff
@@ -47,11 +46,11 @@ export default function Characters() {
   ]
 
   const isOpen = useMemo(() => {
-    return params && !!find(formModeEnum, (value) => value.includes(params[0]))
+    return params && Object.values(formModeEnum).includes(params[0])
   }, [params])
 
   const handleOpenForm = (id) => {
-    const params = id ? [formModeEnum.edit, id] : [formModeEnum.new]
+    const params = id ? [formModeEnum.edit, id] : [formModeEnum.new, null]
```

Comment:
Somehow when I write like this
```
const params = id ? [formModeEnum.edit, id] : [formModeEnum.new]
```
it resulting in
```
console.log(params)
new
console.log(params.length)
3
```
which will not pass the 2nd rule created
```
if (params.length > 2)
```

Reply:
That's because the `pathname` you have provided is not correct, so you are getting unexpected results. The path you provided in `handleCloseForm` is also technically incorrect as well. The code I had provided was only to be used for illustration and not to be copied exactly :wink: . Refer to the next.js docs to figure out what is wrong